### PR TITLE
SEO-2882-Angular-Pivot-table-Maps-NPM

### PR DESCRIPTION
**Pull request by Gowri-V-L**
Hi Aishwarya G,

I have worked in Angular main link, Pivot Table, Maps control pages for the below,

1. Added FT links to the control name in overview content
2. Changed the UTM links as per new valid one
3. Included "Related link" section in Maps and Pivot table page
4. Include "related links" and "About Syncfusion Angular UI Components" section in main page alone
5. Modified all the links with ui in its URL for angular, react, vue to currently valid URLs

Target keyword worked : 
For Pivot control - Angular Pivot Table
For Maps control - Angular Maps Library

Regards,
Gowri V L

### CODE REVIEW

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are Fine.

File: components/maps/README.md [hunk 1]
```diff
@@ -16,58 +18,67 @@ npm install @syncfusion/ej2-maps
...
-2. [React](https://www.npmjs.com/package/@syncfusion/ej2-react-maps?utm_source=npm&utm_campaign=maps)
-3. [Vue.js](https://www.npmjs.com/package/@syncfusion/ej2-vue-maps?utm_source=npm&utm_campaign=maps)
-4. [ASP.NET Core](https://aspdotnetcore.syncfusion.com/Maps/Default#/material)
-5. [ASP.NET MVC](https://aspnetmvc.syncfusion.com/Maps/Default#/material)
-6. [JavaScript (ES5)](https://www.syncfusion.com/products/javascript/maps)
+1. [Angular](https://www.npmjs.com/package/@syncfusion/ej2-ng-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+2. [React](https://www.npmjs.com/package/@syncfusion/ej2-react-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+3. [Vue.js](https://www.npmjs.com/package/@syncfusion/ej2-vue-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+4. [ASP.NET Core](https://aspdotnetcore.syncfusion.com/Maps/Default#/material?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+5. [ASP.NET MVC](https://aspnetmvc.syncfusion.com/Maps/Default#/material?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
```
Comment (Aishwarya-Ganesan) [inline components/maps/README.md:33]: mvc & core links are broken. Please check that and remove

File: components/maps/README.md [hunk 2]
```diff
@@ -16,58 +18,67 @@ npm install @syncfusion/ej2-maps
...
+* [**Print and Export**](https://ej2.syncfusion.com/angular/demos/#/material/maps/print?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Provides options to print and export the rendered maps for future use.
 
 ## Support
 
 Product support is available for through following mediums.
 
-* Creating incident in Syncfusion [Direct-trac](https://www.syncfusion.com/support/directtrac/incidents?utm_source=npm&utm_campaign=maps) support system or [Community forum](https://www.syncfusion.com/forums/essential-js2?utm_source=npm&utm_campaign=maps).
-* New [GitHub issues](https://github.com/syncfusion/ej2-angular-ui-components/issues).
+* Creating incident in Syncfusion [Direct-trac](https://www.syncfusion.com/support/directtrac/incidents?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) support system or [Community forum](https://www.syncfusion.com/forums/essential-js2?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm).
+* New [GitHub issues](https://github.com/syncfusion/ej2-angular-components/issues?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm).
```
Comment (Aishwarya-Ganesan) [inline components/maps/README.md:64]: https://github.com/syncfusion/ej2-angular-components/issues - this link also broken. Please check.

Comment (Gowri-V-L) [reply components/maps/README.md:64]: fixed

File: components/maps/README.md (hunk 1 above)
Comment (Gowri-V-L) [reply components/maps/README.md:33]: removed

**Comment by Gowri-V-L** [comment]
Hi @Aishwarya-Ganesan  

I have fixed the broken links , please merge it

**Pull request merged**